packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -30,7 +30,6 @@ import { useDialog } from '../common/Dialog'
 import { TokenApprovalDialog } from './TokenApprovalDialog'
 import { WithdrawalConfirmationDialog } from './WithdrawalConfirmationDialog'
 import { DepositConfirmationDialog } from './DepositConfirmationDialog'
-import { LowBalanceDialog } from './LowBalanceDialog'
```

Comment:
Do we want to keep the component?

Reply:
Yes, we're keeping it dormant so that any new design/refactoring/changes to these popups later can be quickly implemented in code.